browser/src/control/Control.TopToolbar.js

```diff
@@ -275,10 +278,15 @@ L.Control.TopToolbar = L.Control.extend({
 				window.hideTooltip(this, e.target);
 			},
 			onRefresh: function(event) {
-				if ((event.target === 'styles' || event.target === 'fonts' || event.target === 'fontsizes') && event.item) {
+				if ((event.item && (event.target === 'styles' || event.target === 'fonts' || event.target === 'fontsizes'))
+					|| (event.item && event.item.invisible == true)) {
```

Comment:
it can be simplified to:
`if (event.item && (event.target === 'styles' || event.target === 'fonts' || event.target === 'fontsizes' || event.item.invisible)`

Reply:
also better to use '===' (strict equal - with type checking) so it will not convert other type values like '0' (string) to true

Reply:
Good pointer and much appreciated!